torch/_inductor/scheduler.py

```diff
@@ -1783,7 +1815,7 @@ def _init(self, nodes: List[ir.Operation]) -> None:
         # in codegen we only use buf0, never buf1
         self.mutation_renames: Dict[str, str] = {}
 
-        self.compute_dependencies()
+        name_to_users = self.compute_dependencies()
```

Comment:
Curious: is this change because we need a snapshot of `name_to_users` that's unchanged by the immediately following steps?

Reply:
Ooops, this is actually no longer needed after some changes. I forgot to remove this.